### PR TITLE
Increase Limit of park-entry price

### PR DESCRIPTION
**Pull request by MaxMallon**
Based on a request in the discord server.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Please do the following:
- Add a changelog entry
- Make the commit message adhere to the [Guidelines](https://github.com/OpenRCT2/OpenRCT2/wiki/Commit-Messages), "Increase Limit" is too vague. "Increase limit of park entry price" would be a better message.

**Comment by MaxMallon** [comment]
> * Add a changelog entry

Isn't this already covered by "- Headline feature: [#10664] New save format with increased limits."?

**Comment by Gymnasiast** [comment]
No. It was possible in SV6 already. Also, we don’t want to check everything under that entry ;)

**Review by Gymnasiast — APPROVED** [review]

**Comment by Gymnasiast** [comment]
Could you rebase this and update the changelog?

**Comment by MaxMallon** [comment]
Wait, so this didn't make it in after all? D: ah man..

File: distribution/changelog.txt [hunk 1]
```diff
@@ -45,6 +46,7 @@
 - Improved: [#16740] Allow staff patrol areas to be defined with individual tiles rather than groups of 4x4.
 - Improved: [#16764] [Plugin] Add hook ‘map.save’, called before the map is saved.
 - Improved: [#16925] The queue length of 1000 guests is lifted, and a warning for too long queues is added instead.
+- Improved: [#16999] The maximum price for the park entry has been raised to 999.
```
Comment (Broxzier) [inline distribution/changelog.txt:49]: Remove this one

File: distribution/changelog.txt [hunk 2]
```diff
@@ -4,6 +4,7 @@
 - Fix: [#17005] Unable to set patrol area for first staff member in park.
 - Fix: [#17073] Corrupt ride window and random crashes when trains have more than 144 cars.
 - Fix: [#17080] “Remove litter” cheat does not empty litter bins.
+- Improved: [#16999] The maximum price for the park entry has been raised to 999.
```
Comment (Broxzier) [inline distribution/changelog.txt:7]:
```suggestion
- Improved: [#16999] The maximum price for the park entry has been raised to £999.
```

**Review by Broxzier — CHANGES_REQUESTED** [review]

File: distribution/changelog.txt (hunk 1 above)
Comment (MaxMallon) [reply distribution/changelog.txt:49]: Why does it need to removed?

Comment (MaxMallon) [reply distribution/changelog.txt:49]: Ah, I hadn't noticed the old one was still there too, my bad

**Review by Broxzier — APPROVED** [review]

**Pull request merged**